### PR TITLE
Append to Pass Kernel Options

### PR DESCRIPTION
**Pull request by BaptisteGerondeau**
Sadly I wasn't able to find how to add something to Jorge's pull request (#3 )... 
Sorry Jorge !

I modified mr_provisioner_machine_provision.py to support kernel opts and added default kernel option as an empty string.

### CODE REVIEW

**Comment by niedbalski** [comment]
@BaptisteGerondeau Please rebase your changes against a single commit (git cherry-pick -i HEAD~5) and then force push to your repo, that way is easier to review.

Also, I don't think this specific change is required (I will address this through https://github.com/Linaro/ansible-role-mr-provisioner/pull/3)

**Comment by niedbalski** [comment]
@BaptisteGerondeau  since https://github.com/Linaro/ansible-role-mr-provisioner/pull/3 has been merged, please rebase your changes with the latest master.  After doing that, I think that this proposal wouldn't be required.

**Comment by BaptisteGerondeau** [comment]
There is still the issue of the set_machine_parameters in library/mr_provisioner_machine_provision.py not taking the kernel_options (and consequently not dumping them in the json parameters).

File: library/mr_provisioner_machine_provision.py [hunk 1]
```diff
@@ -115,8 +115,11 @@ def set_machine_parameters(url, token, machine_id, initrd_id=None,
         parameters['preseed_id'] = preseed_id
     if subarch:
         parameters['subarch'] = subarch
+    if kernel_opts:
+        parameters['kernel_opts'] = kernel_opts
+    else:
```
Comment (danrue) [inline library/mr_provisioner_machine_provision.py]: Why is this else necessary when none of the other parameters use it? Please be consistent with the rest of the options, unless this case is in fact different.

**Comment by danrue** [comment]
This looks like a good change to me except for the one small nit on what looks to me to be an errant 'else'.

**Comment by BaptisteGerondeau** [comment]
Deleted the else statement and made the kernel_opts default to "" instead, in the function declaration.
(Sorry I had completely missed that comment...)

**Pull request merged**